<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="leftBox">
       <property name="title">
        <string>图表</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="rightBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>数据</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="traingle_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>157</width>
             <height>387</height>
            </rect>
           </property>
           <attribute name="label">
            <string>三角形</string>
           </attribute>
           <layout class="QHBoxLayout" name="triangle_h_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
          <widget class="QWidget" name="circle_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>157</width>
             <height>387</height>
            </rect>
           </property>
           <attribute name="label">
            <string>圆形</string>
           </attribute>
           <layout class="QHBoxLayout" name="circle_h_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
          <widget class="QWidget" name="ellipse_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>157</width>
             <height>387</height>
            </rect>
           </property>
           <attribute name="label">
            <string>椭圆</string>
           </attribute>
           <layout class="QHBoxLayout" name="ellipse_h_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
          <widget class="QWidget" name="line_page">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>157</width>
             <height>387</height>
            </rect>
           </property>
           <attribute name="label">
            <string>直线</string>
           </attribute>
           <layout class="QHBoxLayout" name="line_h_layout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="maintoolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenPic"/>
   <addaction name="actEdit_Delete"/>
   <addaction name="actEdit_Color"/>
   <addaction name="actLine_Width"/>
   <addaction name="actReset_Item"/>
   <addaction name="actClear_Screen"/>
   <addaction name="actQuit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actItem_triangle"/>
   <addaction name="actItem_Circle"/>
   <addaction name="actItem_Ellipse"/>
   <addaction name="actItem_Line"/>
  </widget>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/132.bmp</normaloff>:/images/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出本系统</string>
   </property>
  </action>
  <action name="actOpenPic">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/128.bmp</normaloff>:/images/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="toolTip">
    <string>打开图片文件</string>
   </property>
  </action>
  <action name="actItem_Circle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/circle.png</normaloff>:/images/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>圆形</string>
   </property>
   <property name="toolTip">
    <string>添加圆形</string>
   </property>
  </action>
  <action name="actEdit_Delete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/108.bmp</normaloff>:/images/images/108.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除Item</string>
   </property>
  </action>
  <action name="actItem_triangle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/triangle.png</normaloff>:/images/images/triangle.png</iconset>
   </property>
   <property name="text">
    <string>三角形</string>
   </property>
  </action>
  <action name="actEdit_Color">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/color.png</normaloff>:/images/images/color.png</iconset>
   </property>
   <property name="text">
    <string>颜色选择器</string>
   </property>
  </action>
  <action name="actClear_Screen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/clearScreen.png</normaloff>:/images/images/clearScreen.png</iconset>
   </property>
   <property name="text">
    <string>清屏</string>
   </property>
  </action>
  <action name="actItem_Ellipse">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/ellipse.png</normaloff>:/images/images/ellipse.png</iconset>
   </property>
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="actItem_Line">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/line.png</normaloff>:/images/images/line.png</iconset>
   </property>
   <property name="text">
    <string>直线</string>
   </property>
  </action>
  <action name="actLine_Width">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/Linde_widget.png</normaloff>:/images/images/Linde_widget.png</iconset>
   </property>
   <property name="text">
    <string>线条设置</string>
   </property>
   <property name="toolTip">
    <string>线条设置</string>
   </property>
  </action>
  <action name="actReset_Item">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/reset.png</normaloff>:/images/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
